mlflow/langchain/langchain_tracer.py

```diff
@@ -376,19 +377,28 @@ def on_tool_start(
         """Start span for a tool run."""
         if metadata:
             kwargs.update({"metadata": metadata})
+
+        # Input string should be JSON-like (it is json with single quotes)
```

Comment:
what is JSON-like? Can you add an example?

Reply:
sth like `"{'foo': 'bar'}"`

Reply:
can you include that in the comment?

Reply:
Interesting, what causes the single quotes? How come we don't just pass it in as a proper JSON?

Reply:
It's because LangChain does `str(dictionary)` and Python produces single quote:(
https://github.com/langchain-ai/langchain/blob/master/libs/core/langchain_core/tools/base.py#L632